formulas/wick-base/functions/wick-is-apt-available

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# Checks whether Apt and Dpkg type applications are running.
+#
+# Examples
+#
+#   wickIsAptAvailable
+#
+# Returns 0 when no Apt or Dpkg type applications are found running.
+# Returns >0 when an Apt or Dpkg type application is found running.
+wickIsAptAvailable() {
+    pidof dpkg > /dev/null 2>&1 && return 1
+    pidof apt-get > /dev/null 2>&1 && return 2
+    pidof synaptic > /dev/null 2>&1 && return 3
+    pidof apt > /dev/null 2>&1 && return 4
+    pidof aptitude > /dev/null 2>&1 && return 5
+    return 0
```

Comment:
I am not seeing this in the style guide now, but I think `return` statements (at least the one on its own) should be preceded by a newline. Perhaps that should be added to the style guide? I've had it pointed out to me before so I think it is valid...